Assets/_Project/Code/Models/Pytanie.cs

```diff
@@ -0,0 +1,63 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+
+public class Pytanie
+{
+    public string Tresc { get; private set; }
+    [JsonProperty("Poprawne Odpowiedzi", Order = 2)]
+    private List<string> poprawneOdpowiedzi = new List<string>();
+    [JsonProperty("Odpowiedzi", Order = 3)]
+    private Dictionary<char, string> odpowiedzi;
```

Comment:
Jest ok. Trzeba to tylko dostosować do już zrobinych przez testerów pytań. Skontaktuj się z nimi i pozmieniaj nazwy w Jsonie oraz typy tych pól aby pasowały

Reply:
czyli zostaje słownik?

Reply:
Szanowny Panie,  uważam iż to testerzy powinni dostosować się pod mojego Jsona. Pozdrawiam.

Reply:
Pozwól więc, że zapytam. Łatwiej zmienić 4-5 linijek kodu czy ponad 100 pytań? Może jakbyś jeszcze się tym wcześniej zajął i wysłał mielibyśmy o czym rozmawiać. Ty jednak wysłałeś swoje zadanie po terminie, kiedy ci konkretni testerzy wykonali swoją pracę sumiennie i na czas. Więc możesz uważać co chcesz, ale dużo efektywniej będzie teraz lekko zmodyfikować twój kod.

Reply:
Nie wiadomo w jakiej formie napisali te pytania, i czy json będzie mógł je odczytać .Zamiana nazw może nie pomóc

Reply:
Nie wiesz bo nie pytałeś. Jak ich przekonasz do swojego to nie mam problemu, ale ja im nie będę kazał zmieniać wszystkiego bo ty tak chcesz. Napisz do nich, pokażą ci co mają i zdecydujecie sami.

Reply:
Nie chodzi o to że ja tak chcę ale o to że forma zapisu do pliku i odczytu z niego przez jsona jest ściśle okreslona przez co trzymamy się konkretnego szablonu. Testerzy mogliby sobie pisac pytania w dowolnie wybranej przez siebie formie a niewiele by to dało skoro json ich nie może odczytać. Gdybym dostał szablon jakim posługują się testerzy przy pisaniu pytań ( o ile bylłby możliwy do zaimplementowania) to bym się nim kierował, niestety komunikacja pomiędzy grupami pozostawia wiele do życzenia.